src/components/FacebookLoginButton.js

```diff
@@ -0,0 +1,88 @@
+import React, {Component, PropTypes} from 'react';
```

Comment:
created a `FacebookLoginButton` component to handle all our specific logic (what permissions, what parameter to get, etc)